src/Bladeburner/Types.ts

```diff
@@ -1,11 +1,4 @@
-import type { BlackOperation, Contract, GeneralAction, Operation } from "./Actions";
-import type {
-  BladeburnerActionType,
-  BladeburnerBlackOpName,
-  BladeburnerContractName,
-  BladeburnerOperationName,
-  BladeburnerGeneralActionName,
-} from "@enums";
+import { BlackOperation, Contract, GeneralAction, Operation } from "./Actions";
```

Comment:
It is *definitely* the case that Types.ts can only use import type.

Reply:
I'll remove the nice-to-haves from the PR